get-config.js

```diff
@@ -0,0 +1,21 @@
+function readConfig() {
+	try {
+		return require( './config' )();
```

Comment:
> to be backwards compatible we can use `typeof` and…

boo! :wink:

@beaucollins do we have a reason to change it? if we do, I'd rather just force everyone to change it; or, I don't see any compelling reason to be loose here with our input.